src/views/Pages/Register/Register.js

```diff
@@ -0,0 +1,241 @@
+          <InputGroupAddon addonType="prepend">
+            <InputGroupText>
+            <span role="img" aria-label="name">&#9942;</span>
+            </InputGroupText>
+          </InputGroupAddon>
+          <Input type="textarea" id="address" value={address} onChange={this.handleChange} placeholder="Company Address" autoComplete="address" required />
+        </InputGroup>
+      </>
+    }
+
+    return (
+      <Form onSubmit={this.handleSubmit} method="post" id={formId}>
+        <InputGroup className="mb-3">
+          <InputGroupAddon addonType="prepend">
+            <InputGroupText>
+              <i className="fa fa-user"></i>
+            </InputGroupText>
+          </InputGroupAddon>
+          <Input type="text" id="firstname" value={firstname} onChange={this.handleChange} placeholder="First Name" autoComplete="first name" required />
+        </InputGroup>
+        <InputGroup className="mb-3">
+          <InputGroupAddon addonType="prepend">
+            <InputGroupText>
+              <i className="fa fa-user"></i>
+            </InputGroupText>
+          </InputGroupAddon>
+          <Input type="text" id="lastname" value={lastname} onChange={this.handleChange} placeholder="Last Name" autoComplete="last name" required />
+        </InputGroup>
+        <InputGroup className="mb-3">
+          <InputGroupAddon addonType="prepend">
+            <InputGroupText>
+            <span role="img" aria-label="name">&#9742;</span>
+            </InputGroupText>
+          </InputGroupAddon>
+          <Input type="text" id="phonenumber" value={phonenumber} onChange={this.handleChange} placeholder="Phone Number" autoComplete="phone number" required />
+        </InputGroup>
+        <InputGroup className="mb-3">
+          <InputGroupAddon addonType="prepend">
+            <InputGroupText><i className="fa fa-envelope"></i></InputGroupText>
+          </InputGroupAddon>
+          <Input type="text" id="email" value={email} onChange={this.handleChange} placeholder="Email" autoComplete="email" required />
+        </InputGroup>
+        {optInp}
+        <InputGroup className="mb-3">
+          <InputGroupAddon addonType="prepend">
+            <InputGroupText><span role="img" aria-label="name">&#9873;</span></InputGroupText>
+          </InputGroupAddon>
+          <Input type="select" id="country" value={country} onChange={this.handleChange}>
+            <option>Nigeria</option>
+            <option>Ghana</option>
+            <option>Guinea</option>
+            <option>Niger</option>
+          </Input>
+        </InputGroup>
+        <span className="error">{f}</span> <br />
+        <Button type="submit" color="success" block>{u}</Button>
+      </Form>
+    )
+  }
+
+  handleChange(e) {
+    this.setState(
+      {
+        [e.target.id]: e.target.value
+      }
+    )
+  }
+
+
+  handleSubmit(e) {
+    e.preventDefault()
+    let { firstname, lastname, phonenumber, email, gender, country, dateofbirth } = this.state
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['firstname' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)